src/components/Users/UserAvailabilityTab.tsx

```diff
@@ -187,16 +200,18 @@ export default function UserAvailabilityTab({ userData: user }: Props) {
             >
               {t("schedule")}
             </Button>
-            <Button
-              variant={view === "exceptions" ? "outline" : "ghost"}
-              onClick={() => setQParams({ tab: "exceptions" })}
-              className={cn(
-                view === "exceptions" && "shadow",
-                "hover:bg-white text-xs sm:text-sm px-2 md:px-4",
-              )}
-            >
-              {t("exceptions")}
-            </Button>
+            {availableUserId && (
+              <Button
+                variant={view === "exceptions" ? "outline" : "ghost"}
+                onClick={() => setQParams({ tab: "exceptions" })}
+                className={cn(
+                  view === "exceptions" && "shadow",
+                  "hover:bg-white text-xs sm:text-sm px-2 md:px-4",
+                )}
+              >
+                {t("exceptions")}
+              </Button>
+            )}
```

Comment:
let's not hide the exceptions tab;

it'd be better to disable the create exception button instead and instead of showing "No exceptions created", let's say that exceptions can be created only if the condition is satisfied